internal/vulnstore/postgres/gc.go

```diff
@@ -0,0 +1,91 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+const (
+	// GCThrottle sets a limit for the number of update
+	// operations deleted in a single GC run.
+	GCThrottle = 3
```

Comment:
The issue I see is the number of delete operation are (at worst) exponential to the number of update operations. 

I wanted some throttling to not attempt GIANT deletes, since an update operation associates with many vulnerabilities.

From an ops perspective this means GC will happen gradually during normal operation, and since GCFull can be trigged, you can run this at off hours once you know your traffic trends.